layouts/partials/banner.html

```diff
@@ -1,19 +1,13 @@
 <section class="news is-medium flex justify-center">
-  <p class="is-size-3-mobile">
-    <span>
-      {{ $rocketIcon := resources.Get "icons/rocket-icon.svg" }}
-      <img
+    <p class="is-size-3-mobile">
+      <a href="https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.3" class="banner-link">Notation v1.0.0-RC.3 is available</a>
```

Comment:
Since rc.4 is coming, suggest changing the banner to rc.4

Reply:
Agree with @zr-msft . We could send another PR to update it to RC.4 after the RC.4 is out.